Client/Frontend/Fakespot/Views/FakespotHighlightsCardView.swift

```diff
@@ -141,6 +150,36 @@ class FakespotHighlightsCardView: UIView, ThemeApplicable {
         dividerView.backgroundColor = theme.colors.borderPrimary
     }
 
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        guard let viewModel else { return }
+        guard viewModel.isOneHighlightGroupWithTwoReviews else { return }
+        guard let longestReview = viewModel.longestTextFromReviews else { return }
+
+        let highlightLabelWidth = longestReview.width(
+            with: DefaultDynamicFontHelper.preferredFont(
+                withTextStyle: .subheadline,
+                size: UX.titleFontSize,
+                weight: .semibold
+            )
+        )
+
+        let highlightsGroupViewWidth = (bounds.width - UIWindow.safeAreaInsets.left * 2) -
+                                       (highlightGroups.first!.imageHeightConstraint?.constant ?? 24) -
+                                       (2 * UX.contentHorizontalSpace)
+
+        let areMoreThanTwoLines = highlightLabelWidth > highlightsGroupViewWidth
+
+        if areMoreThanTwoLines {
+            contentStackView.addArrangedSubview(dividerView)
+            contentStackView.addArrangedSubview(moreButton)
+        } else {
+            contentStackView.removeArrangedView(dividerView)
+            contentStackView.removeArrangedView(moreButton)
+        }
+        updateHighlights(areMoreThanTwoLines)
```

Comment:
I think it would be better to extract all of the code added in `layoutSubviews` into a method.